src/Confluent.SchemaRegistry/Rest/RestService.cs

```diff
@@ -137,6 +161,9 @@ private async Task<HttpResponseMessage> ExecuteOnOneInstanceAsync(HttpRequestMes
                 }
                 catch (HttpRequestException e)
                 {
+                    // don't retry error responses originating from Schema Registry.
+                    if (e is SchemaRegistryException) { throw; }
+
```

Comment:
where do the below exceptions bubble up to the user? Produce()? Poll?

Reply:
`Produce`